DeadSeaTupperware/Project2-Find-My-Duo
# Feature/index model

## Purpose

Reworked index.js model configuration. 

## Changes

User-message association:
- User can send many messages (line 11)
- Each message is sent a user (line 12)

Chatroom-message association:
- Chatroom has many messages (line 15)
- Messages belong to the Chatroom (line 16)

User-Game association:
- many-to-many relationship through a new table "UserGame"

## Steps to Reproduce or Test

- fetch branch
- verify codebase changes 
- verify new index model definition and export in your local branch with PR req
